### PR TITLE
Added Wagtail and initial model definitions

### PR DESCRIPTION
**Pull request by gsidebo**
#### Pre-Flight checklist

- [x] Migrations
  - [x] Migration is backwards-compatible with current production code
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested
- [x] Settings
  - [x] New settings are documented and present in `app.json`
  - [x] New settings have reasonable development defaults, if applicable
  - [ ] Opened issue for DevOps regarding necessary configuration changes to deployed environments

#### What are the relevant tickets?
N/A

#### What's this PR do?
Adds Wagtail, defines some barebones initial models, and a template tag to allow us to serve Wagtail images without using the Wagtail image view

#### How should this be manually tested?
- Run the `configure_wagtail` management command
- Go to `/cms`, upload an image or two, set an image for the `hero` in the home page, and add a product page as a child of the home page. Make sure you can see the hero image on the home page and that both pages load (they're intentionally _very_ barebones)

### CODE REVIEW

**Review by rhysyngsun — CHANGES_REQUESTED** [review]
A few small things on the code, how should i go about testing this?

**Comment by gsidebo** [comment]
@rhysyngsun Just added testing steps

File: README.md [hunk 1]
```diff
@@ -7,174 +7,33 @@ mitX Online
...
 `createsuperuser` command**.
 
-# Optional Setup
+### Configure xPro and Open edX
 
-Described below are some setup steps that are not strictly necessary
-for running the app
+1. See [Configure Open edX](docs/configure_open_edx.md)
+1. If one doesn't already exist, add an alias to `/etc/hosts` for Open edX. We have standardized this alias
+  to `edx.odl.local`. Your `/etc/hosts` entry should look like this:
```
[the remark this answers is not in the document]
Comment (rhysyngsun) [reply README.md]: well this one was embarassingly wrong

File: main/views.py [hunk 2]
```diff
@@ -13,3 +14,8 @@ def index(request):
         request,
         "index.html",
     )
+
+
+def cms_signin_redirect_to_site_signin(request):
+    """Redirect wagtail admin signin to site signin page"""
+    return redirect_to_login(reverse("wagtailadmin_home"), login_url="/signin")
```
Comment (rhysyngsun) [inline main/views.py:21]: `reverse()` isn't imported so this is throwing an error, it could probably use a test too.

**Review by rhysyngsun — CHANGES_REQUESTED** [review]
One issue but i think otherwise it's looking good

**Review by rhysyngsun — APPROVED** [review]

**Pull request merged**